src/clue.js

```diff
@@ -2,25 +2,184 @@
 
 // Suspects Array
 
-const suspectsArray = [];
+const suspectsArray = [
```

Comment:
In this case there's no need to assign a name to the objects ( "MrGreen = { }" , "drOrchid = { }" , ... ):

```
const suspectsArray = [
  {
    firstName: "Jacob",
    lastName: "Green",

    (...)

  },
  {
    firstName: "Doctor",
    lastName: "Orchid",

    (...)

  },

    (...)
```